tests/framework/CodeInterfaceTests/raven_running_raven_internal_models/test_rom_trainer.xml

```diff
@@ -198,7 +198,7 @@
         <Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
         <Output>CladTempThreshold,time</Output>
     </HistorySet>
-    <PointSet inputTs="2" name="outputMontecarloRomND">
+    <PointSet name="outputMontecarloRomND">
```

Comment:
I see several changes in the way we define the DataObjects. Do we need the conversion scripts?

Reply:
I think for this is not needed because I just removed a left over from the inputs (now it is 
strictly checked by the InputData). It is better to keep it because the user, in case it has ```inputTs``` on purpose, it  needs to know that that flag was deactivated since years.

Reply:
ok, good to me.